hsengine/game.h

```diff
@@ -485,10 +485,10 @@ public slots:
      * board control value function.
      */
     struct BoardControlScore {
-        static const float TOUGH_MINIONS_MULT = 2.5; //!< Multiplier for the "tough minions" factor
-        static const float HAND_MULT = 1;            //!< Multiplier for the "hand" factor
-        static const float MINIONS_MULT = 1.75;      //!< Multiplier for the "minions number" factor
-        static const float SITUATION_MULT = 3.75;    //!< Multiplier for the "situation" factor
+        static constexpr const float TOUGH_MINIONS_MULT = 2.5; //!< Multiplier for the "tough minions" factor
```

Comment:
I just didn't know about Q_DECL_CONSTEXPR, but it is probably a good idea to use it instead of constexpr... and I've just push the change to github (and this version works with gcc).

Note that gcc require either constexpr or Q_DECL_CONSTEXPR to compile this code.